### ISSUE TITLE
A8 not recognized

### ISSUE THREAD

**Issue by Sharpvx** [foxlet/bootra1n#93]
Bootra1n 0.11.0 i686
iPhone 6+ on iOS 12.4.8 once plugged in checkra1n still says connect your iPhone, tried multiple cables, ports, computers with no luck.

**Comment by DEALWITHIT199** [comment]
Only A9(X) devices are supported Bootra1n 0.11.0. use the Bootra1n 0.10.2.

**Comment by DEALWITHIT199** [comment]
Did you connect the cable to a USB 3.0 Slot and USB Boot Drive on 2.0 USB Slot?

**Comment by Sharpvx** [comment]
I did but got the same issue

**Comment by DEALWITHIT199** [comment]
try using a high quality cable

**Comment by Sharpvx** [comment]
Used 3 original Apple cables, same issue

**Comment by Gr8z** [comment]
+1 

Can confirm that iPhone 6s+ is not recognized in 0.11.

**Comment by DEALWITHIT199** [comment]
> Bootra1n 0.11.0 i686
> iPhone 6+ on iOS 12.4.8 once plugged in checkra1n still says connect your iPhone, tried multiple cables, ports, computers with no luck.

Try turning on Safe Mode under options and re connect your iDevice and see if it works

**Comment by Gr8z** [comment]
> Try turning on Safe Mode under options and re connect your iDevice and see if it works

Tried that just now, no go.

**Comment by Sharpvx** [comment]
> > Try turning on Safe Mode under options and re connect your iDevice and see if it works
> 
> Tried that just now, no go.

+1

**Comment by derpyranbowpop** [comment]
Same Here. Also have noticed my device dosent charge or detect that its been pluged in

**Comment by Sharpvx** [comment]
I wonder if it’s an issue with Checkra1n or Bootra1n itself.

**Comment by DEALWITHIT199** [comment]
Did you use rufus to make the USB Boot Drive?

**Comment by DEALWITHIT199** [comment]
> > Try turning on Safe Mode under options and re connect your iDevice and see if it works
> 
> Tried that just now, no go.

Did you make the USB Bootable Drive using rufus?

**Comment by DEALWITHIT199** [comment]
remember guys Bootra1n 0.11.0 is still pre-release (its like beta)

**Comment by Windgale** [comment]
This may be a stupid question to ask but does 0.11.0 support iPhone X? It is apparently an A11.

I only ask because I was able to use 0.10.2 and 0.10.1 to jailbreak my iPhone but 0.11.0 won’t even acknowledge that I plugged it in, on two different computers using different USB combinations.

**Comment by DEALWITHIT199** [comment]
> This may be a stupid question to ask but does 0.11.0 support iPhone X? It is apparently an A11.
> 
> I only ask because I was able to use 0.10.2 and 0.10.1 to jailbreak my iPhone but 0.11.0 won’t even acknowledge that I plugged it in, on two different computers using different USB combinations.

no

**Comment by Windgale** [comment]
> no

Ah...  that would explain my failure over the last couple hours lol.

Thank you clarifying!

**Comment by Gr8z** [comment]
> Did you make the USB Bootable Drive using rufus?

Yes, the same setup as the previous working version. Tested wit both iOS 13 and 14. iPhone 6s+ is not recognized.

**Comment by DEALWITHIT199** [comment]
> This may be a stupid question to ask but does 0.11.0 support iPhone X? It is apparently an A11.
> 
> I only ask because I was able to use 0.10.2 and 0.10.1 to jailbreak my iPhone but 0.11.0 won’t even acknowledge that I plugged it in, on two different computers using different USB combinations.

I will be supported in the future. i hope so

**Comment by DEALWITHIT199** [comment]
> > Did you make the USB Bootable Drive using rufus?
> 
> Yes, the same setup as the previous working version. Tested wit both iOS 13 and 14. iPhone 6s+ is not recognized.

what version of bootra1n are you using?

**Comment by Chibyqt** [comment]
Must be a Bootra1n issue. I have the same issue with my iPad air 2 when i plug it in it just doesnt get detected by Checkra1n so probaby some drivers are missing?! With an older Bootra1n version its working but i need the newest one for iOS 14. so please fix <3

**Comment by carlgo18** [comment]
+1 
My iPad Air 2 also isn’t recognized. Getting usb: device descriptor read/64, error -71

**Comment by DEALWITHIT199** [comment]
> Must be a Bootra1n issue. I have the same issue with my iPad air 2 when i plug it in it just doesnt get detected by Checkra1n so probaby some drivers are missing?! With an older Bootra1n version its working but i need the newest one for iOS 14. so please fix <3



> +1
> My iPad Air 2 also isn’t recognized. Getting usb: device descriptor read/64, error -71

Use a High Quality cable and turn on safe mode under options in checkra1n

**Comment by DEALWITHIT199** [comment]
GUYS. Everyone who is having problem that your phone is not being recognized do the following:

1. Make sure you boot to the USB Drive in legacy mode not UFEI Mode.
2. If your phone is still not recognized by checkra1n. “reconnect” your device and see if a message appears on your phone saying “Trust this computer” and click on trust.

**Comment by DEALWITHIT199** [comment]
> GUYS. Everyone who is having problems that your phone is not being recognized do the following:
> 
> 1. Make sure you boot to the USB Drive in legacy mode not UFEI Mode.
> 2. If your phone is still not recognized by checkra1n. “reconnect” your device and see if a message appears on your phone saying “Trust this computer” and click on trust.

**Comment by carlgo18** [comment]
> > Must be a Bootra1n issue. I have the same issue with my iPad air 2 when i plug it in it just doesnt get detected by Checkra1n so probaby some drivers are missing?! With an older Bootra1n version its working but i need the newest one for iOS 14. so please fix <3
> 
> > +1
> > My iPad Air 2 also isn’t recognized. Getting usb: device descriptor read/64, error -71
> 
> Use a High Quality cable and turn on safe mode under options in checkra1n



> GUYS. Everyone who is having problems that your phone is not being recognized do the following:
> 
> 1. Make sure you boot to the USB Drive in legacy mode not UFEI Mode.
> 2. If your phone is still not recognized by checkra1n. “reconnect” your device and see if a message appears on your phone saying “Trust this computer” and click on trust.

- Original Apple Cable ✅
- Legacy Mode (already did this back when 0.9.8)✅
- tried reconnecting multiple times, shows no “trust this computer” prompt

**Comment by asrgaohifd** [comment]
This is definitely a bug, I was using 0.10.2 with no issues at all. Then I updated it and now the USB isn’t being detected.

**Comment by DEALWITHIT199** [comment]
> > > Must be a Bootra1n issue. I have the same issue with my iPad air 2 when i plug it in it just doesnt get detected by Checkra1n so probaby some drivers are missing?! With an older Bootra1n version its working but i need the newest one for iOS 14. so please fix <3
> > 
> > 
> > > +1
> > > My iPad Air 2 also isn’t recognized. Getting usb: device descriptor read/64, error -71
> > 
> > 
> > Use a High Quality cable and turn on safe mode under options in checkra1n
> 
> > GUYS. Everyone who is having problems that your phone is not being recognized do the following:
> > 
> > 1. Make sure you boot to the USB Drive in legacy mode not UFEI Mode.
> > 2. If your phone is still not recognized by checkra1n. “reconnect” your device and see if a message appears on your phone saying “Trust this computer” and click on trust.
> 
> * Original Apple Cable ✅
> * Legacy Mode (already did this back when 0.9.8)✅
> * tried reconnecting multiple times, shows no “trust this computer” prompt

Use checkn1x because bootra1n is still beta and checkn1x is better and small file size

**Comment by DEALWITHIT199** [comment]
Guys go to this [link](https://github.com/foxlet/bootra1n/issues/95) iPhone not recognized solved!

**Comment by Hackinman120** [comment]
I wonder if this could be another solution.

・Putting your iPhone in recovery mode(Use Windows iTunes)
・Then run bootra1n.

**Comment by DEALWITHIT199** [comment]
> I wonder if this could be another solution.
> 
> ・Putting your iPhone in recovery mode(Use Windows iTunes)
> ・Then run bootrain.

Theres already a recovery mode inluded in bootra1n checkra1n

**Comment by Hackinman120** [comment]
> > I wonder if this could be another solution.
> 
> > 
> 
> > ・Putting your iPhone in recovery mode(Use Windows iTunes)
> 
> > ・Then run bootrain.
> 
> 
> 
> Theres already a recovery mode inluded in bootra1n checkra1n


Instead of using bootra1n to go into recovery mode
That if you put it in recovery mode in iTunes and then bootra1n, it might be recognized.

I was recognized for it.

**Comment by DEALWITHIT199** [comment]
> > > I wonder if this could be another solution.
> > 
> > 
> > > 
> > 
> > 
> > > ・Putting your iPhone in recovery mode(Use Windows iTunes)
> > 
> > 
> > > ・Then run bootrain.
> > 
> > 
> > Theres already a recovery mode inluded in bootra1n checkra1n
> 
> Instead of using bootra1n to go into recovery mode
> That if you put it in recovery mode in iTunes and then bootra1n, it might be recognized.
> 
> I was recognized for it.

Wow so there are no errors during jailbreak?

**Comment by DEALWITHIT199** [comment]
> > > I wonder if this could be another solution.
> > 
> > 
> > > 
> > 
> > 
> > > ・Putting your iPhone in recovery mode(Use Windows iTunes)
> > 
> > 
> > > ・Then run bootrain.
> > 
> > 
> > Theres already a recovery mode inluded in bootra1n checkra1n
> 
> Instead of using bootra1n to go into recovery mode
> That if you put it in recovery mode in iTunes and then bootra1n, it might be recognized.
> 
> I was recognized for it.

Good Solution. but I prefer using Checkn1x for now because checkn1x is small file size and stable rather bootra1n has a lot of bugs and its in beta.
